tests/conftest.py

```diff
@@ -224,7 +224,7 @@ def pytest_generate_tests(metafunc: pytest.Metafunc) -> None:
         selected_constructors = [
             x
             for x in selected_constructors
-            if x not in GPU_CONSTRUCTORS and x not in "modin"  # too slow
+            if x not in GPU_CONSTRUCTORS and x != "modin"  # too slow
```

Comment:
Just to double check, did we just want to remove `modin` or was it a typo to remove anything that has modin in the name `"modin" not in x`?

Just checking because I added `modin[pyarrow]` above

Reply:
probably wanted to remove modin - thanks!